CafeRecords/Pages/Index.cshtml

```diff
@@ -1,32 +1,59 @@
 ﻿@page
 @model IndexModel
 @{
-    ViewData["Title"] = "Home page";
-    CafeRecord.Cafe[] allCafes = (CafeRecord.Cafe[])ViewData["allCafes"];
-    OwnerRecord.Owner[] allOwners = (OwnerRecord.Owner[])ViewData["allOwners"];
+    
     List<OwnerRecord.Owner> cafeOwners = (List<OwnerRecord.Owner>)ViewData["cafeOwners"];
 }
+    <html>
+    <head>
+        <style>
+            body {
+                background-image: url("https://desktopwalls.net/wp-content/uploads/2014/07/Chicago%20Colorful%20Skyline%20View%20Desktop%20Wallpaper.jpg");
```

Comment:
added a background image relevant to the project data.

Reply:
Styling is nice.  Consider putting it into a linked stylesheet, so that this page is cleaner, and it can be used in other files as well.